src/gazebo2rviz/gazebo2moveit_collision_node.py

```diff
@@ -48,16 +58,24 @@ def on_model_states_msg(model_states_msg):
             sdf2moveit.delete_collision_object(modelinstance_name)
             del model_cache[modelinstance_name]
 
+
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('-f', '--freq', type=float, default=2, help='Frequency Markers are published (default: 2 Hz)')
     args = parser.parse_args(rospy.myargv()[1:])
 
     rospy.init_node('gazebo2moveit')
-    
+
     global sdf2moveit
     sdf2moveit = Sdf2moveit()
 
+    global get_object_type_service
```

Comment:
In general, using global is most of the time bad practice. We should really just re-write this as a class instead of spreading globals randomly all over the place. @beatrizleon do you think we should do that within this PR, do it later, or leave it as it is for now?